on_device_training/mobilevit/mobilevit_console/mobilevit_console/Program.cs

```diff
@@ -0,0 +1,168 @@
+﻿using Microsoft.ML.OnnxRuntime;
+using Microsoft.ML.OnnxRuntime.Tensors;
+
+namespace mobilevit_console
+{
+    public class Program
+    {
+        static string PARENTDIR = Directory.GetParent(Directory.GetCurrentDirectory()).Parent.Parent.Parent.Parent.FullName;
+
+        static string CHECKPOINTPATH = Path.Combine(PARENTDIR, "checkpoint");
+        static string TRAININGMODELPATH = Path.Combine(PARENTDIR, "training_model.onnx");
+        static string EVALMODELPATH = Path.Combine(PARENTDIR, "eval_model.onnx");
+        static string OPTIMIZERMODELPATH = Path.Combine(PARENTDIR, "optimizer_model.onnx");
+
+        static string TRAINEDMODELFILE = "trained_mobilevit.onnx";
+
+        static string DEFAULTINFERENCE1 = Path.Combine(PARENTDIR, "test1.png");
+        static string DEFAULTINFERENCE2 = Path.Combine(PARENTDIR, "test3.png");
+        
+        static void Main(string[] args)
+        {
+            if (args.Length < 1)
```

Comment:
Is it possible to provide an argument helper for the  user since this is a console app?